kbchat/kbchat_test.go

```diff
@@ -3,40 +3,40 @@ package kbchat
 import (
 	"crypto/rand"
 	"encoding/hex"
-	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
 )
 
-type Bots struct {
-	Alice1   *OneshotOptions
-	Alice2   *OneshotOptions
-	Bob1     *OneshotOptions
-	Charlie1 *OneshotOptions
+type team struct {
```

Comment:
making it lowercase still pollutes the `kbchat` namespace with something that really only necessary for testing right here. also, this isn't actually a team. it's a combo of a team name and a topic name. i think i wrote this before, but there's a lot of confusion in these bots libraries about what is a `channel`. i would argue, and i think we should push towards this as much as possible, that a `channel` is the combination of a `team name` and a `topic`. so i might call this combo a `ChatChannel`. and try to use it across the entire app for this combo. maybe there's already something that already does this.

Reply:
can we just use an existing channel type?